packages/remix-cloudflare-pages/worker.ts

```diff
@@ -58,7 +58,7 @@ export function createPagesFunctionHandler<Env = any>({
     try {
```

Comment:
Let's wrap the attempt at serving static files in a `if (["GET", "HEAD"].includes(request.method)) {` to prevent post request bodies from being locked.